person.rb

```diff
@@ -1,21 +1,25 @@
 require './nameable'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] The :rental needs to be getter and setter as well, please add it under attr_accessor

Reply:
@GzimAsani,
thanks for your good and valuable feedback,
I have considered your comment into account.